prac_05/kivy_layout.kv

```diff
@@ -0,0 +1,20 @@
+# This is a Kv file.
+# It contains the declarations of the widgets that will be displayed.
+BoxLayout:
+    orientation: 'vertical'
+    Label:
+        text: app.status_text
+        font_size: 30
+        size_hint_y: 0.2
+    BoxLayout:
+        orientation: 'horizontal'
+        Button:
+            text: "Up"
+            on_press: app.handle_press(1)
+        Button:
+            text: "Down"
+            on_press: app.handle_press(-1)
+    BoxLayout:
+        id: names_box
+    Label:
+        text: "I did this :)"
```

Comment:
Hey just noticed that you missed part of the question about resizing the label to only take up 10% of the window space